### ISSUE TITLE
Training Image Transformer

### ISSUE THREAD

**Issue by sahajgarg** [tensorflow/tensor2tensor#1665]
### Description

I'm unable to replicate unconditional image generation results using the image transformer with 1D local attention reported to have bits/dim of 2.90. When training a model with categorical variables and the base hparams defined in the code, the training curve is below. How do I replicate the results in the paper?

![image](https://user-images.githubusercontent.com/16550324/63287855-2e465e00-c270-11e9-9f8f-a3e293a44234.png)

### Command to reproduce

```
t2t-trainer \
  --generate_data \
  --data_dir=~/t2t/data \
  --output_dir=~/tmp/model/test \
  --problem=image_cifar10_plain_gen_rev \
  --model=imagetransformer \
  --hparams_set=imagetransformer_cifar10_base \
  --train_steps=100000 \
  --eval_steps=1000
```

### Environment

```
OS: ubuntu 16.04

$ pip freeze | grep tensor
mesh-tensorflow==0.0.5
-e git+https://github.com/tensorflow/tensor2tensor.git@63c015f964c1166d181d8efd232abd856574fd83#egg=tensor2tensor
tensorboard==1.14.0
tensorflow==1.14.0
tensorflow-datasets==1.1.0
tensorflow-estimator==1.14.0
tensorflow-gpu==1.14.0
tensorflow-hub==0.5.0
tensorflow-metadata==0.14.0
tensorflow-probability==0.7.0


$ python -V
Python 3.5.2
```
